test/integration/helpers/serverspec/xpack_spec.rb

```diff
@@ -7,7 +7,9 @@
     it { should_not contain "path.conf: /etc/elasticsearch/#{vars['es_instance_name']}" }
     it { should contain "path.data: /var/lib/elasticsearch/localhost-#{vars['es_instance_name']}" }
     it { should contain "path.logs: /var/log/elasticsearch/localhost-#{vars['es_instance_name']}" }
-    it { should contain 'xpack.security.enabled: false' }
-    it { should contain 'xpack.watcher.enabled: false' }
+    if vars.key?('es_xpack_features')
```

Comment:
this conditional doesn't seem to work, the 2 following tests are absents of https://devops-ci.elastic.co/job/elastic+ansible-elasticsearch+pull-request/47/OS=ubuntu-1804,TEST_TYPE=xpack-upgrade,VERSION=7.x,label=linux/console even if test with the same conditional in https://github.com/jmlrt/ansible-elasticsearch/blob/627a29ae093285adc3818214b63edb36955368d6/test/integration/helpers/serverspec/shared_spec.rb#L38-L47 is present.